client/client.go

```diff
@@ -141,7 +141,7 @@ func NewClientWithConfig(config *Config) (*Client, error) {
 		// if enabled we log all requests and responses to sterr
 		client.httpClient.Logger = log.New(os.Stderr, "", log.LstdFlags)
 		client.httpClient.ResponseLogHook = func(l retryablehttp.Logger, resp *http.Response) {
-			l.Printf("[DEBUG] Response: %s %s", resp.Request.Method, resp.Request.URL.String())
```

Comment:
I changed this to say "Request" because we weren't actually logging much about the response, just the request. I added the request ID and the status as well.